Hestia/app/src/main/java/hestia/backend/Activator.java

```diff
@@ -56,4 +56,27 @@ public String toString() {
     public void setState(ActivatorState state) {
         this.state = state;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        Activator activator = (Activator) o;
+
+        if (activatorId != activator.activatorId) return false;
+        if (!state.equals(activator.state)) return false;
```

Comment:
true. but this is needed to signal a fireChangeEvent(). I suggest not to remove it now.